scripts/test_aktualizr.sh

```diff
@@ -0,0 +1,15 @@
+UPDATE_ID=$(curl "https://app.atsgarage.com/api/v1/multi_target_updates' -H 'Cookie: sizeVerifyHidden=1; fireworksPageAcknowledged=1; systemReady=1; PLAY_SESSION=$PLAY_SESSION" -H 'Origin: https://app.atsgarage.com' -H 'Accept-Encoding: gzip, deflate, br' -H 'Accept-Language: en-US,en;q=0.9,uk;q=0.8,de;q=0.7' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36' -H 'Content-Type: application/json;charset=UTF-8' -H 'Accept: application/json, text/plain, */*' -H 'Csrf-Token: c7ae20514d7f5e791a7e39a763720d21224c8d7c-1518391691761-d060f16175da1bbef90e3a38' -H 'Referer: https://app.atsgarage.com/' -H 'Connection: keep-alive' -H 'DNT: 1' --data-binary '{"targets":{"patriotyk-laptop":{"to":{"target":"selfupdate_2.0","checksum":{"method":"sha256","hash":"1fc3289584041fb01b7022a7c37da8bac3f92fdf58d6d27114d0ad93d135cb34"},"targetLength":1753258},"targetFormat":null,"generateDiff":false}}}' --compressed)
+UPDATE_ID=$(sed -e 's/^"//' -e 's/"$//' <<<"$UPDATE_ID")
+echo "UPDATE_ID: $UPDATE_ID"
+curl "https://app.atsgarage.com/api/v1/admin/devices/385b8a0c-0add-4b43-b153-41728b2a4ced/multi_target_update/46205055-daf6-4fe3-a1eb-7cbcdf4ea960' -X PUT -H 'Cookie: sizeVerifyHidden=1; fireworksPageAcknowledged=1; systemReady=1; PLAY_SESSION=$PLAY_SESSION" -H 'Origin: https://app.atsgarage.com' -H 'Accept-Encoding: gzip, deflate, br' -H 'Accept-Language: en-US,en;q=0.9,uk;q=0.8,de;q=0.7' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36' -H 'Accept: application/json, text/plain, */*' -H 'Csrf-Token: c7ae20514d7f5e791a7e39a763720d21224c8d7c-1518391691761-d060f16175da1bbef90e3a38' -H 'Referer: https://app.atsgarage.com/' -H 'Connection: keep-alive' -H 'Content-Length: 0' -H 'DNT: 1' --compressed
+
+aktualizr -c /persistent/config_selfupdate.toml --poll-once
```

Comment:
So the goal here is to launch aktualizr, fetch a specific update from the server, then start a new aktualizr to verify that the download worked? I'm a bit confused where the installation is happening. I think I might recommend naming this script something like `test_aktualizr_selfupdate` just to be a bit more clear what is really being tested.

Reply:
Installation haapnes when we run `aktualizr -c /persistent/config_selfupdate.toml --poll-once`
If You ask about the installation of aktualizr that performs selfupdate, this is just builded debian package that we build for testing in docker. To save running tests time I added this tests to script that tests debian package build